src/write.c

```diff
@@ -3090,6 +3090,18 @@ static avifResult avifRWStreamWriteProperties(avifItemPropertyDedup * const dedu
             // We don't add an 'lsel' property since many decoders do not support it and will reject the image,
             // see https://github.com/AOMediaCodec/libavif/pull/2429
         }
+        for (size_t i = 0; i < itemMetadata->numProperties; i++) {
+            avifItemPropertyDedupStart(dedup);
+            avifImageItemProperty * prop = &(itemMetadata->properties[i]);
```

Comment:
Nit: Omit the parentheses. This kind of expression (`&foo->bar[i]`) is very common in libavif.

Reply:
Fixed